pkg/proxy/util/network.go

```diff
@@ -25,7 +25,7 @@ import (
 // for testing purposes.
 type NetworkInterfacer interface {
 	Addrs(intf *net.Interface) ([]net.Addr, error)
-	Interfaces() ([]net.Interface, error)
```

Comment:
As far as I can tell `Interfaces()` and `Addrs()` are still used in winkernel.  Once you fix that, you should be able to remove both functions here (leaving just the new one).  I think?

Reply:
seems we are good, it is not used in windows
```
grep -r NetworkInterfacer pkg/
pkg/proxy/util/testing/fake.go:// FakeNetwork implements the NetworkInterfacer interface for test purpose.
pkg/proxy/util/testing/fake.go:// Addrs is part of NetworkInterfacer interface.
pkg/proxy/util/testing/fake.go:// Interfaces is part of NetworkInterfacer interface.
pkg/proxy/util/network.go:// NetworkInterfacer defines an interface for several net library functions. Production
pkg/proxy/util/network.go:type NetworkInterfacer interface {
pkg/proxy/util/network.go:// RealNetwork implements the NetworkInterfacer interface for production code, just
pkg/proxy/util/network.go:// Addrs wraps net.Interface.Addrs(), it's a part of NetworkInterfacer interface.
pkg/proxy/util/network.go:// Interfaces wraps net.Interfaces(), it's a part of NetworkInterfacer interface.
pkg/proxy/util/network.go:var _ NetworkInterfacer = &RealNetwork{}
pkg/proxy/util/utils.go:// NetworkInterfacer is injected for test purpose.
pkg/proxy/util/utils.go:func GetNodeAddresses(cidrs []string, nw NetworkInterfacer) (sets.String, error) {
pkg/proxy/iptables/proxier.go:  networkInterfacer utilproxy.NetworkInterfacer
pkg/proxy/ipvs/proxier.go:      networkInterfacer     utilproxy.NetworkInterfacer
```

and the `NetworkInterfacer` seems to be only used as parameter of `GetNodeAddresses` that is the one we are just changing.
```
pkg/proxy/ipvs/proxier.go:              nodeAddrSet, err := utilproxy.GetNodeAddresses(proxier.nodePortAddresses, proxier.networkInterfacer)
pkg/proxy/iptables/proxier.go:  networkInterfacer utilproxy.NetworkInterfacer
pkg/proxy/iptables/proxier.go:          networkInterfacer:        utilproxy.RealNetwork{},
pkg/proxy/iptables/proxier.go:  nodeAddresses, err := utilproxy.GetNodeAddresses(proxier.nodePortAddresses, proxier.networkInterfacer)
```